random-beans/src/main/java/io/github/benas/randombeans/api/EnhancedRandom.java

```diff
@@ -73,13 +73,28 @@
     /**
      * Generate a random instance of the given type.
      *
-     * @param type the target class type
+     * @param type           the target class type
      * @param excludedFields the name of fields to exclude
-     * @param <T>  the target type
+     * @param <T>            the target type
      * @return a random instance of the given type
      */
     public static <T> T random(final Class<T> type, final String... excludedFields) {
         return aNewEnhancedRandomBuilder().build().nextObject(type, excludedFields);
     }
 
+    /**
+     * Provides static access to {@link EnhancedRandom#objects(Class, String...)}
```

Comment:
This is not exactly the same, since the `EnhancedRandom#objects(Class, String...)` returns an infinite stream and not a bounded one (with 100 elements). I think this method can be removed and keep the user choose the size of the stream with `randoms(final Class<T> type, final int streamSize, final String... excludedFields)`. what do you think?